hanami-devtools.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |spec| # rubocop:disable Metrics/BlockLength
   spec.add_dependency "codecov", "~> 0.1"
   spec.add_dependency "excon", "~> 0.60"
   spec.add_dependency "poltergeist", "~> 1.17"
-  spec.add_dependency "rack", "~> 2.0"
+  spec.add_dependency "rack", "~> 3.1"
```

Comment:
I think we want this to be `">= 2.0"` too, so that other Hanami gems can use both Rack 2 and 3 as part of a test matrix.